src/Adapter/Category/QueryHandler/GetCategoryForEditingHandler.php

```diff
@@ -71,6 +73,25 @@ public function handle(GetCategoryForEditing $query)
             );
         }
 
+        /**
+         * Select recursivly the subcategories in one SQL request
+         *
+         * @see https://dev.mysql.com/doc/refman/8.0/en/with.html
```

Comment:
Is it a mysql 8 only feature ? If yes, I'm afraid we cant use it 😢

Reply:
Oh my gad, I checked only for this
![image](https://user-images.githubusercontent.com/1462701/59105094-702f5b80-8933-11e9-8799-d3f346250954.png)
And thinking everything was ok :sob: :sob:

Reply:
Done! Thanks a lot